core/src/main/scala/org/broadinstitute/workbench/hamm/api/HammRoutes.scala

```diff
@@ -0,0 +1,81 @@
+package org.broadinstitute.workbench.hamm.api
+
+
+import cats.effect._
+import io.circe.generic.auto._
+import org.broadinstitute.workbench.hamm.service._
+import org.http4s.HttpRoutes
+import org.http4s.headers.{Allow, Authorization}
+import org.http4s.server.Router
+import org.broadinstitute.workbench.hamm.HammLogger
+import org.broadinstitute.workbench.hamm.auth.SamAuthProvider
+import org.broadinstitute.workbench.hamm.model.{HammException, JobId, WorkflowId}
+import org.http4s.Credentials.Token
+import org.http4s.circe._
+import org.http4s.dsl.Http4sDsl
+import org.http4s._
+import org.http4s.server.middleware.Logger
+import org.http4s.syntax.kleisli._
+
```

Comment:
Http4s routes class. Each call to the service is wrapped in an IO. 
I just replaced all the `F`s with `IO`s since our use of it is limited and we're really never going to use anything but IO there.